<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Adsorption Model Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="concentration_of_absorbate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="concentration_of_absorbent">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="input3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C0</string>
    </property>
   </widget>
   <widget class="QLabel" name="input4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cs</string>
    </property>
   </widget>
   <widget class="QLabel" name="input1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="time">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="amount_of_absorbate">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="input2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>qt</string>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_plot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result_data">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="method">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>851</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter experimental data in the boxes, spliting with ',' and starting from time 0.
For t and qt, start a new line if there are multiple datasets.</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_unit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="qt_unit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>mg g-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="C0_unit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mg L-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cs_unit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>g L-1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
